packages/url_launcher/url_launcher_web/lib/src/link.dart

```diff
@@ -70,28 +104,37 @@ class WebLinkDelegateState extends State<WebLinkDelegate> {
     return Stack(
       fit: StackFit.passthrough,
       children: <Widget>[
-        widget.link.builder(
-          context,
-          widget.link.isDisabled ? null : _followLink,
+        Semantics(
+          link: true,
+          identifier: semanticIdentifier,
+          value: widget.link.uri?.getHref(),
+          child: widget.link.builder(
+            context,
+            widget.link.isDisabled ? null : _followLink,
+          ),
         ),
         Positioned.fill(
-          child: PlatformViewLink(
-            viewType: linkViewType,
-            onCreatePlatformView: (PlatformViewCreationParams params) {
-              _controller = LinkViewController.fromParams(params);
-              return _controller
-                ..setUri(widget.link.uri)
-                ..setTarget(widget.link.target);
-            },
-            surfaceFactory:
-                (BuildContext context, PlatformViewController controller) {
-              return PlatformViewSurface(
-                controller: controller,
-                gestureRecognizers: const <Factory<
-                    OneSequenceGestureRecognizer>>{},
-                hitTestBehavior: PlatformViewHitTestBehavior.transparent,
-              );
-            },
+          child: ExcludeFocus(
```

Comment:
It's the other way around. The children's semantics are kept intact (with the addition of `Semantics(link: true, ...)` around them).

What I'm excluding is the platform view that's injected by the Link widget. Platform views are leaf nodes, so there's no way for us to place the user-supplied children inside the platform view (maybe in the future we could use multi-view to render the children inside the platform view?).

The issue with the current approach (prior to the PR) is that the semantics of the link are split in 2: (1) the platform view (i.e. `<a href="...">`), (2) the semantics nodes generated by the children (e.g. button and/or text). This leads to a poor a11y experience because the `<a>` element doesn't have any label or other info, and the children (e.g. button) don't have any link information (e.g. href). I had to come up with a way to merge the two somehow. Since I'm in control of the platform view, I know how to replicate its semantics in the Flutter world.

Obviously, I'm no semantics expert, so there may be a better approach that I'm missing. Happy to discuss if you have ideas.